x-pack/plugins/enterprise_search/public/applications/shared/cypress/commands.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/*
+ * Shared non-product-specific commands
+ */
+
+/*
+ * Log in a user via XHR
+ * @see https://docs.cypress.io/guides/getting-started/testing-your-app#Logging-in
+ */
+interface Login {
+  username?: string;
+  password?: string;
+}
+export const login = ({
```

Comment:
I mentioned this in Slack, but just to reiterate: the reason why we're not using `Cypress.commands.add()` or creating `cy.login` (ent-search) as opposed to plain fns and `import { login } from '...'` in here is because of Typescript(™️).

It essentially requires a custom `*.d.ts` file with definitions for every custom command (https://docs.cypress.io/guides/tooling/typescript-support#Types-for-custom-commands) and I just didn't want to bother with it. If someone else wants to down the road, please feel free!

For comparison, the Security team does both - they have a [handful of cy.() commands](https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/cypress/support/index.d.ts) but [way more manually imported actions](https://github.com/elastic/kibana/tree/master/x-pack/plugins/security_solution/cypress/tasks) which they filed under 'tasks' (I left ours under support/commands).